app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -147,7 +152,7 @@
       conceptGoals: ->
         return @level.goals.filter((g) => g.concepts?.length)
       conceptGoalsCompleted: ->
-        return @conceptGoals.filter((g) => @session.state.goalStates[g.id].status is 'success').length
+        return @conceptGoals.filter((g) => @session.toJSON().state.goalStates[g.id].status is 'success').length
```

Comment:
Why do we need to add toJSON() here?

Reply:
Good catch!! I need to revert this back to what it was earlier.
 I checked the code to remember why I had made this change. I had changed the propsData in courseVictoryModal.coffee to just pass session, instead of @session.toJSON() for some reason and I had reverted it back. I have missed reverting this back here. I will fix this.